addons/account/static/src/components/many2x_tax_tags/many2x_tax_tags.js

```diff
@@ -56,14 +58,28 @@ export class Many2ManyTaxTagsAutocomplete extends Many2XAutocomplete {
 export class Many2ManyTaxTagsField extends Many2ManyTagsField {
     static components = {
         ...Many2ManyTagsField.components,
-        Many2XAutocomplete: Many2ManyTaxTagsAutocomplete,
+        Many2XAutocomplete: Many2XTaxTagsAutocomplete,
+    };
+}
+
+export class Many2OneTaxTagsField extends Many2OneField {
+    static components = {
+        ...Many2OneField.components,
+        Many2XAutocomplete: Many2XTaxTagsAutocomplete,
     };
 }
 
 export const many2ManyTaxTagsField = {
     ...many2ManyTagsField,
     component: Many2ManyTaxTagsField,
-    additionalClasses: ['o_field_many2many_tags']
+    additionalClasses: ['o_field_many2many_tags'],
```

Comment:
I would just remove this comma, you're adding it here which prevents future blame breaking (what should probably have been done originally), but you're doing it by breaking previous blames :smiley:

Reply:
Or not as it was @yosuanicolaus that proposed it. I just don't really find it useful